tests/test_format_and_parse.py

```diff
@@ -1080,3 +1080,19 @@ def test_190(self):
         expected_sql = "SELECT typeof(sum(a3)) FROM a GROUP BY a1"
         expected_json = {'from': 'a', 'select': {'value': {'typeof': {'sum': 'a3'}}}, 'groupby': {'value': 'a1'}}
         self.verify_formatting(expected_sql, expected_json)
+
+    @skip("The escape function does not recognize the backticks,"
```

Comment:
Good idea to skip the tests!  If someone works on a MySQL formatter, they have something to start with.